src/streaming/MediaPlayer.js

```diff
@@ -270,6 +278,8 @@ MediaPlayer = function (context) {
                     doAutoPlay.call(this);
                 }
             }
+
+            this.adapter.reset();
```

Comment:
@AkamaiDASH @KozhinM This line is causing some problems for me.  We are resetting the manifest adapter after we call doAutoPlay().  This causes the adapter to be reset after it has been initialized with data from the new manifest.  Can I move this line into the `if (!resetting)` block so that it is only reset when the reset of the system objects are reset?